.github/workflows/continuous-integration.yml

```diff
@@ -24,6 +24,7 @@ jobs:
           - php-versions: 5.4
             dependency-levels: 'lowest'
             experimental: false
+      fail-fast: false
```

Comment:
Maybe this change to the workflow it's what's stopping the Action execution?

If you really need this, make a separate PR, so I can approve and merge it immediately.

Reply:
Well, `fail-fast` causes all jobs to run, even if one of that fails. My reasoning for adding this would be that we are probably interested in seeing all failures, rather than just one, especially since we build on such a wide range of PHP versions.

The idea was to add it at least temporarily, so that there's no need for me to wait until CI finishes again just to see another error. I therefore have no objections to removing this before this PR gets merged, if it is allowed here temporarily. At the same time, perhaps the need for it proves that it's useful, and it could be worth keeping. Entirely up to you.

Reply:
Due to current policies protecting Actions right now, changing the workflow is a issue that stops jobs from starting. I would revert it for now, and discuss it at a later date.

Reply:
Done. Please see #3.